dockerfiles/Dockerfile.tensorrt

```diff
@@ -26,6 +26,6 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
     cp onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt &&\
     cp onnxruntime/ThirdPartyNotices.txt /code/ThirdPartyNotices.txt &&\
     cd onnxruntime &&\
-    /bin/sh ./build.sh --parallel --skip_submodule_sync --cuda_home /usr/local/cuda --cudnn_home /usr/lib/x86_64-linux-gnu/ --use_tensorrt --tensorrt_home /workspace/tensorrt --config Release --build_wheel --update --build --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) "'CMAKE_CUDA_ARCHITECTURES="${CMAKE_CUDA_ARCHITECTURES}"'" &&\
+    /bin/sh ./build.sh --parallel --skip_submodule_sync --cuda_home /usr/local/cuda --cudnn_home /usr/lib/x86_64-linux-gnu/ --use_tensorrt --tensorrt_home /workspace/tensorrt --config Release --build_wheel --update --build --cmake_extra_defines ONNXRUNTIME_VERSION=$(cat ./VERSION_NUMBER) '"CMAKE_CUDA_ARCHITECTURES='${CMAKE_CUDA_ARCHITECTURES}'"' &&\
```

Comment:
why does it need so many quotes.
see https://github.com/microsoft/onnxruntime/blob/91c15843cdb8074e219510007cba0bb0e560a29b/tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml#L56 for reference on Linux?

Reply:
Because ${CMAKE_CUDA_ARCHITECTURES} is a variable (i.e. 75 for perf test on T4): 
1. 'CMAKE_CUDA_ARCHITECTURES=${CMAKE_CUDA_ARCHITECTURES}' evaluates to 'CMAKE_CUDA_ARCHITECTURES=${CMAKE_CUDA_ARCHITECTURES}' not 'CMAKE_CUDA_ARCHITECTURES=75' 
2. Previously "'CMAKE_CUDA_ARCHITECTURES="${CMAKE_CUDA_ARCHITECTURES}"'" evaluated to  'CMAKE_CUDA_ARCHITECTURES=75' but it still took a long time (session creation 100s). 
3. It seems '"CMAKE_CUDA_ARCHITECTURES='${CMAKE_CUDA_ARCHITECTURES}'"' evaluates to "CMAKE_CUDA_ARCHITECTURES=75" and session creation is down to 3s.